app/src/main/java/org/schabi/newpipe/fragments/list/BaseListFragment.java

```diff
@@ -372,6 +372,9 @@ protected void showStreamDialog(final StreamInfoItem item) {
         if (KoreUtil.shouldShowPlayWithKodi(context, item.getServiceId())) {
             entries.add(StreamDialogEntry.play_with_kodi);
         }
+
+        entries.add(StreamDialogEntry.show_channel_details);
```

Comment:
I realized this should be shown only if there is a channel attached to this item (this is the case for YouTube, but it may not be the case for other services. Only add the channel option if `!isNullOrEmpty(item.getUploaderUrl())`